dpl-docs/views/layout.dt

```diff
@@ -113,6 +113,8 @@ html(lang='en-US')
         h1 <code><a href="#{info.linkTo(info.mod)}">#{info.mod.qualifiedName}</a>.#{title[0 .. $-24]}</code> #{title[$-24 ..$]}
       - else if (hidx > 0 && title[0 .. hidx] != "Module" && info.mod)
         h1 #{title[0 .. hidx]} <code><a href="#{info.linkTo(info.mod)}">#{info.mod.qualifiedName}</a>.#{title[hidx+1 .. $]}</code>
+      - else if (hidx > 0 && title[0 .. hidx] == "Module" && info.mod)
```

Comment:
> What the limit of stuff usable in these .dt documents?

With limits you mean D code? It's all converted and compiled as normal D code.
The template engine is very similar to jade http://jade-lang.com/reference/.